### ISSUE TITLE
Training against gradients, at least for equilibrium geometries (pull gradients toward zero)

### ISSUE THREAD

**Issue by jminuse** [proteneer/khan#53]

**Comment by proteneer** [comment]
This is probably doable but we'd need to

1) selectively apply this to only the equilibrium geometries (which means we'd need to be able to mark them as such)
2) determine the loss function/norm on the vectors (cosine distance, etc.)
3) call tf.gradients explicitly to generate the batched gradients.

I'd actually prefer we generate a dataset with all force information anyways for the sake of consistency.

**Comment by jminuse** [comment]
We could just compare the gradients component-wise, or if we're comparing to zero, just take the magnitude of the force vector as the loss.

**Comment by proteneer** [comment]
I'll start taking a stab at this.

**Comment by proteneer** [comment]
It looks like we have a number of ways to address including fuzzing and explicitly training to gradients, is it okay to close this @jminuse
